tests/SConscript

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConscript.tests()
```

Comment:
Add `pyList=[]` as argument for `tests()`. Also add a `setup.cfg` to enable flake8 testing.